pkg/debug/Dockerfile

```diff
@@ -29,25 +30,28 @@ ENV LSHW_VERSION 02.19.2
 ENV OPENSSH_VERSION 9.8p1
 
 # setting up building account
-RUN adduser -G abuild -D builder || :
-RUN su builder -c 'abuild-keygen -a -n'
+RUN abuild-keygen -a -n
 
-COPY --chown=builder:abuild abuild/ /
+RUN mkdir -p /root/packages
+COPY abuild/ /
 ENV MUSL_VERSION v1.2.3
 ENV DISTFILES_MIRROR /tmp/mirror
-ADD --chown=builder:abuild https://git.musl-libc.org/cgit/musl/snapshot/${MUSL_VERSION}.tar.gz ${DISTFILES_MIRROR}/musl-${MUSL_VERSION}.tar.gz
-RUN su builder -c 'cd /musl && abuild checksum && abuild -r'
+ADD https://git.musl-libc.org/cgit/musl/snapshot/${MUSL_VERSION}.tar.gz ${DISTFILES_MIRROR}/musl-${MUSL_VERSION}.tar.gz
+WORKDIR /musl
+RUN abuild -F checksum
+RUN abuild -F -r
 
+RUN find / -name musl-1.2.3-r0.apk
```

Comment:
What is the purpose of this find command? Forgot from debug?

Reply:
you can use an approach like here: https://github.com/lf-edge/eve/blob/master/pkg/cross-compilers/Dockerfile#L19 to fetch the whole index beforehand... but I'm wondering why we didn't experience this error before...

Reply:
you can also try `apk add --no-network ....`